Dockerfile.arm64

```diff
@@ -14,5 +14,6 @@
 FROM arm64v8/alpine:3.7
 LABEL maintainer "Casey Davenport <casey@tigera.io>"
 
-ADD bin/kube-controllers-linux-amd64 /usr/bin/kube-controllers
+ADD bin/kube-controllers-linux-arm64 /usr/bin/kube-controllers
```

Comment:
So, I should have pointed this out earlier - how much larger does adding this extra binary make the container?

The k8s go repos are pretty big. Ideally we'd do something like this:

```
/bin/kube-controllers --check-readiness
```

So, you can use the same binary to both run the daemon and also check the daemon's readiness. That means we get one statically compiled thing, which is probably half the size of two.

That said, if it's only a few MB then no need to over optimize - LMK how much bigger this makes the container.

Reply:
It increases the image size from 55MB to 58.4MB.

Reply:
Ok that's not much at all. Let's leave it as is then!